OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -1278,6 +1278,5 @@ function Register-Tentacle {
         $registerArguments += @("--tenanted-deployment-participation", $TenantedDeploymentParticipation)
     }
 
-    Write-Verbose "Registering with arguments: $registerArguments"
```

Comment:
This is the main fix - this line is redundant and doesn't pass the arguments through the scrubber.